drydock/manifest_builder/infrastructure/tutor_based_manifests.py

```diff
@@ -43,16 +43,23 @@ def __init__(self, options: dict) -> None:
             f"{self.template_root}/kustomization.yml",
     ]
 
+    def _get_hook_context_app(self, app_name):
+        if int(tutor_env.__version__.split(".")[0]) > 16:
```

Comment:
Shouldn't it be > 15?.

Can we use something like [`packaging.version.parse`](https://packaging.pypa.io/en/latest/version.html#packaging.version.parse)?

```python
from packaging import version
# ...
        tutor_version = version.parse(tutor_env.__version__)
        if tutor_version > version.parse("15"):
```